api/tests/shared/unit/domain/services/token-service_test.js

```diff
@@ -86,6 +86,28 @@ describe('Unit | Shared | Domain | Services | Token Service', function () {
     });
   });
 
+  describe('#createAccessTokenForSaml', function () {
+    it('should return a valid json web token', function () {
+      // given
+      const userId = 123;
+      const source = 'external';
+      sinon.stub(settings.authentication, 'secret').value('a secret');
+      sinon.stub(settings.authentication, 'accessTokenLifespanMs').value(1000);
+      const accessToken = 'valid access token';
+      const audience = 'https://app.pix.fr';
+      const firstParameter = { user_id: userId, source, aud: audience };
+      const secondParameter = 'a secret';
+      const thirdParameter = { expiresIn: 1 };
+      sinon.stub(jsonwebtoken, 'sign').withArgs(firstParameter, secondParameter, thirdParameter).returns(accessToken);
```

Comment:
Des petites suggestions pour rendre ce test plus lisible, même si j'ai bien vu qu'il s'agit d'une copie d'un test existant. Et du coup je serais assez pour que ce test existant soit mis à jour pour être plus lisible, mais dans un commit `test(api): blablabla` final.

```suggestion
    it('returns a valid json web token', function () {
      // given
      const secret = 'a secret';
      const userId = 123;
      const source = 'external';
      
      sinon.stub(settings.authentication, 'secret').value(secret);
      sinon.stub(settings.authentication, 'accessTokenLifespanMs').value(1000);

      const accessToken = 'valid access token';
      const audience = 'https://app.pix.fr';
      const payload = { user_id: userId, source, aud: audience };
      const secretOrPrivateKey = secret;
      const options = { expiresIn: 1 };
      sinon.stub(jsonwebtoken, 'sign').withArgs(payload, secretOrPrivateKey, options).returns(accessToken);
```